doc/make.py

```diff
@@ -21,6 +21,14 @@
 import webbrowser
 import jinja2
 
+sys.path.insert(0, os.path.abspath('../scripts'))
```

Comment:
I'm -1 on importing anything in `scripts/` from other places. When we make changes to the scripts, we don't need to worry about breaking other parts of the code at the moment. I like to keep it this way.

I think this part is not essential, is it?